test/jasmine/tests/gl2d_plot_interact_test.js

```diff
@@ -925,8 +925,8 @@ describe('Test scattergl autorange:', function() {
                 marker: {size: ms}
             }])
             .then(function() {
-                expect(gd._fullLayout.xaxis.range).toBeCloseToArray([-0.05, 1.05], 'x range');
-                expect(gd._fullLayout.yaxis.range).toBeCloseToArray([-0.11, 1.11], 'y range');
+                expect(gd._fullLayout.xaxis.range).toBeCloseToArray([-0.119, 1.119], 2, 'x range');
+                expect(gd._fullLayout.yaxis.range).toBeCloseToArray([-0.199, 1.199], 2, 'y range');
```

Comment:
In comparison, at `1e5 - 1` pts (i.e. below the big-data threshold), we get:

![image](https://user-images.githubusercontent.com/6675409/37040298-7f4f737c-2127-11e8-8819-d1c2ba9a1214.png)


@alexcjohnson I'll reference https://github.com/plotly/plotly.js/issues/2417 - but honestly feel free to close it if you think `Axes.expand` is satisfactory now.

Reply:
Right, so I guess that result is saying that in this case the [constant pixel padding we choose](https://github.com/plotly/plotly.js/pull/2437/commits/3875fcc1d767af30fb5e8e08a9548191d754c70e#diff-e03c5a2a46f9392088513c32645ea9d5R107) is a bit bigger than it would need to be... but this is a random case, a lot of real cases will have correlations, like all the biggest markers are near one edge, so I think this is a pretty good compromise.

Reply:
> so I think this is a pretty good compromise.

Yep. I think so too :ok_hand: